us.yaml

```diff
@@ -19,60 +94,273 @@ months:
+    wday: 5
+  - name: Lyndon Baines Johnson Day # fixed
+    regions: [us_tx]
+    mday: 27
   9:
   - name: Labor Day
     week: 1
     regions: [us]
     wday: 1
+  - name: Rosh Hashanah
+    regions: [us_tx]
+    function: rosh_hashanah(year)
+  - name: Yom Kippur
+    regions: [us_tx]
+    function: yom_kippur(year)
   10:
   - name: Columbus Day
     week: 2
-    regions: [us]
+    regions: [us_al, us_az, us_co, us_ct, us_dc, us_ga, us_id, us_il, us_in, us_ma, us_md, us_me, us_mo, us_mt, us_ne, us_nj, us_nm, us_ny, us_oh, us_pa, us_ri, us_ut, us_va, us_wv]
     wday: 1
+  - name: Columbus Day
+    week: 2
+    regions: [us_ak, us_ar, us_ca, us_de, us_fl, us_hi, us_mi, us_mn, us_nd, us_nv, us_or, us_sd, us_tx, us_vt, us_wa, us_wi, us_wy]
+    type: informal
+    wday: 1
+  - name: Alaska Day # fixed
+    regions: [us_ak]
+    mday: 18
+  - name: Nevada Day # Last Friday in October
+    regions: [us_nv]
+    week: -1
+    wday: 5
   11:
+  - name: Election Day
+    regions: [us_de, us_hi, us_il, us_in, us_mt, us_nj, us_ny, us_pa, us_ri]
+    function: election_day(year)
   - name: Veterans Day
     regions: [us]
     mday: 11
     observed: to_weekday_if_weekend(date)
   - name: Thanksgiving
-    week: 4
     regions: [us]
+    week: 4
     wday: 4
-  - name: Day after Thanksgiving
+  - name: Family Day
     function: day_after_thanksgiving(year)
-    regions: [us_ca]
+    regions: [us_nv]
+  - name: State Holiday
+    function: day_after_thanksgiving(year)
+    regions: [us_ga]
+  - name: Presidents' Day
+    function: day_after_thanksgiving(year)
+    regions: [us_nm]
+  - name: Lincoln's Birthday
+    function: day_after_thanksgiving(year)
+    regions: [us_in]
+  - name: American Indian Heritage Day
+    function: day_after_thanksgiving(year)
+    regions: [us_md]
+  - name: Day after Thanksgiving (Black Friday)
+    function: day_after_thanksgiving(year)
+    regions: [us_ca, us_de, us_fl, us_ia, us_il, us_ks, us_ky, us_me, us_mi, us_mn, us_ms, us_ne, us_nh, us_nc, us_pa, us_sc, us_ok, us_tn, us_tx, us_va, us_wa, us_wv]
   12:
+  - name: Christmas Eve
+    regions: [us_ar, us_mi, us_nc, us_sc, us_tx, us_wi]
+    mday: 24
+  - name: Christmas Eve (Holiday) # Note: Always if Christmas Eve don't match
+    regions: [us_mi, us_sc, us_va]
+    mday: 24
+    function: christmas_eve_holiday(date)
   - name: Christmas Day
     regions: [us]
     mday: 25
-    observed: to_weekday_if_weekend(date)
+    observed: to_monday_if_weekend(date)
+  - name: Day after Christmas
+    regions: [us_ar, us_nc, us_ok, us_sc, us_tn, us_tx]
+    mday: 26
+  - name: New Year's Eve
+    regions: [us_mi, us_wi]
+    mday: 31
 methods:
+  christmas_eve_holiday:
+    arguments: date
+    source: |
+      beginning_of_month = Date.civil(date.year, date.month, 1)
+      (date.saturday? || date.sunday?) ? date.downto(beginning_of_month).find {|date| date if date.wday == 5} : date
+  rosh_hashanah:
+    arguments: year
+    source: |
+      rosh_hashanah_dates = {
```

Comment:
@ppeble:

Yeah, I agree with you. It's similar to Ramadan and Sacrifice Feast in the Turkish calendar (see #55). In addition to a calculation we need a transformation from the Hebrew into the Gregorian calendar to generate the dates for Rosh Hashanah and Yom Kippur. 

https://en.wikipedia.org/wiki/Jewish_and_Israeli_holidays_2000–2050